client/ftp.js

```diff
@@ -0,0 +1,29 @@
+import Client from 'basic-ftp';
+
+export class FTPClient {
+  constructor(timeout, logger, encoding) {
+    this.client = new Client(timeout);
+    this.client.ftp.encoding = encoding || this.client.ftp.encoding;
+    this.client.ftp.log = logger;
+  }
+
+  async size(path) {
+    return this.client.size(path);
+  }
+
+  async access(options = {}) {
+    return this.client.access(options);
+  }
+
+  async downloadTo(destination, fromRemotePath, startAt = 0) {
+    return this.client.downloadTo(destination, fromRemotePath, startAt);
+  }
+
+  close() {
+    this.client.close();
+  }
+
+  get closed() {
```

Comment:
Is this method used anywhere?

Reply:
Is used on the implementation.

https://github.com/Thomas-Smyth/ftp-tail/blob/3be20cfbe95707360236e585b0001e3d8c4181fb/index.js#L152-L159